packages/contact-center/cc-components/src/components/task/CallControl/CallControlCustom/consult-transfer-popover.tsx

```diff
@@ -1,143 +1,235 @@
+              selectedCategory === 'Dial Number' ? 'consult-category-button-active' : ''
+            }`}
           >
-            Queues
-          </TabNext>
-          <TabNext key={DIAL_NUMBER} className="dial-number-tab" active={selectedTab === DIAL_NUMBER}>
             Dial Number
-          </TabNext>
-        </TabListNext>
-      )}
-
-      {/* If both are empty, show the big empty state */}
-      {!showTabs && <ConsultTransferEmptyState message={getEmptyStateMessage(selectedTab, showTabs)} />}
-
-      {/* If agents tab is selected and empty */}
-      {showTabs && selectedTab === AGENTS && noAgents && (
-        <ConsultTransferEmptyState message={getEmptyStateMessage(selectedTab, showTabs)} />
-      )}
+          </Button>
+        )}
+        {showEntryPointTab && (
+          <Button
+            variant={selectedCategory === 'Entry Point' ? 'primary' : 'secondary'}
+            size="small"
+            onClick={handleEntryPointClick}
+            className={`consult-category-button-wide ${
+              selectedCategory === 'Entry Point' ? 'consult-category-button-active' : ''
+            }`}
+          >
+            Entry Point
+          </Button>
+        )}
+      </div>
 
-      {/* If queues tab is selected and empty */}
-      {showTabs && selectedTab === QUEUES && noQueues && (
-        <ConsultTransferEmptyState message={getEmptyStateMessage(selectedTab, showTabs)} />
-      )}
+      {!hasAnyData && <ConsultTransferEmptyState message="No data available for consult transfer." />}
 
-      {/* Render lists if not empty */}
-      {showTabs &&
-        selectedTab === AGENTS &&
+      {selectedCategory === 'Agents' &&
         !noAgents &&
         renderList(
-          filteredAgents,
-          (agent) => agent.agentId,
-          (agent) => agent.agentName,
-          (id, name) => {
-            handleAgentSelection(id, name, onAgentSelect, logger);
-          }
+          buddyAgents.map((agent) => ({id: agent.agentId, name: agent.agentName})),
+          (item) => handleAgentSelection(item.id, item.name, onAgentSelect, logger)
         )}
 
-      {showTabs &&
-        selectedTab === QUEUES &&
-        !noQueues &&
-        renderList(
-          filteredQueues,
-          (queue) => queue.id,
-          (queue) => queue.name,
-          (id, name) => {
-            handleQueueSelection(id, name, onQueueSelect, logger);
-          }
-        )}
-      {showTabs && selectedTab === DIAL_NUMBER && (
-        <DialNumberUI
-          title="Dial Number"
-          buttonIcon={buttonIcon}
-          onButtonPress={(dialNumber) => {
-            onDialNumberSelect(dialNumber);
-          }}
-          logger={logger}
-        />
+      {selectedCategory === 'Queues' && !noQueues && (
+        <div>
+          {renderList(
+            queuesData.map((q) => ({id: q.id, name: q.name})),
+            (item) => handleQueueSelection(item.id, item.name, onQueueSelect, logger)
+          )}
```

Comment:
Ideally we want to simplify this all we should see in presentational is 
```suggestion
          {renderList(queuesData)}
```
We can process the data in utils

Reply:
As mentioned above - this convolutes things and adds extra complexity to this PR... perhaps we can look at it later on.